app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -177,6 +177,11 @@ def handle_telephony_result(method:, default:)
           otp_make_default_number: default,
           reauthn: reauthn?,
         )
+      elsif @telephony_result.error.is_a?(Telephony::OptOutError) &&
+            IdentityConfig.store.sms_resubscribe_enabled
+        flash.delete(:info)
```

Comment:
Not immediately clear to me, what flash is this removing?

Reply:
Testing locally:
- Sign in to the account page with an existing account
- Add a new phone MFA
- Enter the `OPT_OUT_PHONE_NUMBER` (2255559999)
- ... this flash pops up:

<img width="691" alt="Screen Shot 2022-02-04 at 10 38 20 AM" src="https://user-images.githubusercontent.com/458784/152584572-df961dc5-5272-4a04-99ca-fa2729e62c12.png">

Reply:
Just realized the masked phone number is the wrong one .... gonna look into that 🙈

Reply:
@anniehirshman-gsa had recently mentioned that alert as something where maybe it won't continue around (in its current form) for much longer -- could we include a comment linking to where it's assigned so that it's obvious to clean up if the alert is removed?

Reply:
Sure, how about 080d82fd6641da1fbdff8949f5112c0851e4530c ?

Reply:
> Sure, how about [080d82f](https://github.com/18F/identity-idp/commit/080d82fd6641da1fbdff8949f5112c0851e4530c) ?

Perfect, thanks 👍